_gsocprojects/2019/project_AllenSIMTtoSPMD.md

```diff
@@ -0,0 +1,10 @@
+---
+title: SIMT to SPMD code translation
```

Comment:
This is completely up to you, but the project page will look nicer without this title, it will anyway appear on the list of proposals.